assets/test/specs/form.spec.js

```diff
@@ -32,6 +40,8 @@ describe('Form Validation', function () {
     jasmine.getFixtures().fixturesPath = 'base/specs/fixtures/';
     loadFixtures('form-fixture.html');
     form = require('../../javascripts/validation/form.js');
+    toggleDynamic = require('../../javascripts/modules/toggleDynamicFormFields.js');
+    document.body.classList.add("js")
```

Comment:
This class actually needs to be added to the fixture container, not the body, sorry:

`document.getElementById(jasmine.getFixtures().containerId).classList.add('js');`